idd/Energy+.idd.in

```diff
@@ -1734,6 +1734,140 @@ Site:GroundTemperature:Deep,
        \type real
        \default 16
 
+Site:GroundTemperature:Undisturbed:FiniteDifference,
+       \memo Undisturbed ground temperature object using a
+       \memo detailed finite difference 1-D model
+       \min-fields 7
+  A1,  \field Name
+       \required-field
```

Comment:
@mmatts Need a \reference UndisturbedGroundTempModels (or similar) on the Name field in all three of the new objects.
 Then all of the "Undisturbed Ground Temperature Model Name" fields need \object-list UndisturbedGroundTempModels